test/ServiceBus.Tests/SlowConsumingTests/EHSlowConsumingTests.cs

```diff
@@ -119,9 +118,9 @@ public EHSlowConsumingTests(Fixture fixture)
             var healthyConsumers = await SetUpHealthyConsumerGrain(this.fixture.GrainFactory, streamId.Guid, StreamNamespace, StreamProviderName, healthyConsumerCount);
 
             //set up producer and start producing
-            var producer = this.fixture.GrainFactory.GetGrain<ISampleStreaming_ProducerGrain>(Guid.NewGuid());
+            var producer = this.fixture.GrainFactory.GetGrain<ITypedProducerGrainProducingInt>(Guid.NewGuid());
             await producer.BecomeProducer(streamId.Guid, StreamNamespace, StreamProviderName);
-            await producer.StartPeriodicProducing();
+            await producer.StartPeriodicProducing(TimeSpan.FromMilliseconds(100));
 
```

Comment:
decrease producer producing frequency, which will help healthy consumers to catch up faster after slow consumer stopped reading, which is another aspect this test is testing